tests/test_with_hypothesis.py

```diff
@@ -0,0 +1,15 @@
+from hypothesis.strategies import binary
+
+import pytest
+
+from cryptography.fernet import Fernet
+
+
+hypothesis = pytest.importorskip("hypothesis")
```

Comment:
I think that's a relic from when hypothesis did not have 2.6 support. I'll
remove that when I get to a computer.

On Wed, 30 Sep 2015 at 8:15 AM Alex Gaynor <notifications@github.com> wrote:

> In tests/test_with_hypothesis.py
> <https://github.com/pyca/cryptography/pull/1773#discussion_r40746860>:
>
> > @@ -0,0 +1,15 @@
> > +from hypothesis.strategies import binary
> > +
> > +import pytest
> > +
> > +from cryptography.fernet import Fernet
> > +
> > +
> > +hypothesis = pytest.importorskip("hypothesis")
>
> When would this import fail, but the first one succeed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/1773/files#r40746860>.
>